tabbycat/options/preferences.py

```diff
@@ -1334,3 +1335,34 @@ class PointsEmailLinkText(StringPreference):
     section = email
     name = 'team_points_email_link_text'
     default = "To consult the current team standings, visit:"
+
+# ==============================================================================
+account_creation = Section('accounts', verbose_name=_('Account Creation'))
+# ==============================================================================
+
+
```

Comment:
Adding and blanking the fields are a bit awkward; could there be a toggle "activate" switch or a way to automatically close the keys?

Reply:
Yea, I think this needs some form of toggle to enable. By default with a blank key the signup page is enabled to users at `/accounts/signup/%20/`

Reply:
Damn I should have noticed that - I'll add a toggle

Reply:
I've had a change in mind about this; I'm fine with blanking/adding strings rather than having toggles. @philipbelesky , blank keys are treated as `False` and so would give permission denied now.